x-pack/plugins/monitoring/public/components/logstash/pipeline_listing/pipeline_listing.js

```diff
@@ -159,7 +159,6 @@ export class PipelineListing extends Component {
               sorting={sortingOptions}
               message={upgradeMessage}
               pagination={pagination}
-              fetchMoreData={fetchMoreData}
```

Comment:
don't look for fetchModataData explicitly because its removed in the react version now